api/src/main/java/io/knotx/fragments/api/FragmentResult.java

```diff
@@ -70,6 +77,17 @@ public static FragmentResult fail(Fragment fragment, String errorCode, String er
         FragmentOperationFailure.newInstance(errorCode, errorMessage));
   }
 
+  // Should be used only by caller to indicate synchronous or asynchronous exception
```

Comment:
Since this is a comment explaining how the method should be used, maybe it should be a javadoc